kernel/dtree.mli

```diff
@@ -41,7 +41,7 @@ val pp_matching_problem : Format.formatter -> matching_problem -> unit
 (** Type of decision trees *)
 type dtree =
   | Switch  of int * (case*dtree) list * dtree option (** Switch [i] [(case_0,tree_0) ; ... ; (case_n, tree_n)] [tree_opt] test if the [i] arg of a pattern can be match with one of the case of the list. if it does then look at the corresponding tree, otherwise, look at the default tree *)
-  | Test    of matching_problem * constr list * Term.term * dtree option (** Test [pb] [cstrs] [te] [tree_opt] are the leaves of the tree. Check that each problem can be solves and such that constraints are satisfied. If it does then return a local context for the term [te]. *)
+  | Test    of rule_name * matching_problem * constr list * Term.term * dtree option (** Test [pb] [cstrs] [te] [tree_opt] are the leaves of the tree. Check that each problem can be solves and such that constraints are satisfied. If it does then return a local context for the term [te]. *)
```

Comment:
Documentation doesn't account for the new argument of Test constructor.
Also it should probably not start at the same line but rather be its own paragraph underneath.